### ISSUE TITLE
Select - Axe Dev tool violation - ARIA attributes must conform to valid values

### ISSUE THREAD

**Issue by satheeshkumarp-sage** [Sage/carbon#5507]
### Current behaviour

Ensures all ARIA attributes have valid values.
This has been detected on Axe Dev Tools to check for accessibility requirements

Element Source:
<input aria-invalid="false" aria-describedby="" autocomplete="off" aria-expanded="false" role="combobox" name="da6eae19-c2f3-8baa-50f3-9c5c1f967144" type="text" id="68c621ae-e26b-ba8e-5519-bbb3346b2f33" data-element="input" class="sc-cHGsZl MsCrg" value="2022/2023" aria-activedescendant="bcba1c7f-01b3-f855-c098-539efc3f537b">

Fix the following:
Invalid ARIA attribute value: aria-activedescendant="bcba1c7f-01b3-f855-c098-539efc3f537b"

![image](https://user-images.githubusercontent.com/44157880/193012566-a6bf0763-2180-417b-92e5-8ec0445a6060.png)

### Expected behaviour

Ensures all ARIA attributes have valid values and No error appears when scanning using Axe Dev Tools

No accessibility violations should occur.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/docs/select--default-story
https://carbon.sage.com/v/109.2.2/index.html?path=/docs/select--default-story

### JIRA Ticket (Sage Only)

SKPU-7075

### Carbon Version

109.2.2

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by edleeks87** [comment]
For anyone triaging this, you will need to select a value then re-open the list and run the axe extension over it, the issue is only present in 109.2.2 and not in latest vesions

**Comment by edleeks87** [comment]
Hi @satheeshkumarp-sage as far as I can tell this issue is not present in the latest versions of Carbon. Is there anything preventing you from upgrading from 109.2.2 to latest (111.0.0)?

**Comment by edleeks87** [comment]
Closing as fixed from v110 onwards
